### ISSUE TITLE
Add cheat option to resurrect dead pilots

### ISSUE THREAD

**Issue by dasrehzockt** [dcs-liberation/dcs_liberation#1274]
since the DCS AI can lead to massive losses, even though you got a very strong advantage in every aspect, can be very frustrating. No matter what i do, after a round i ALWAYS have losses on my own side, that should not have happened. until 3.0 you could just cheat money to cover the losses, wait 1 round and bam. there you go as if nothing has ever happened. however, if a pilot is dead now, he is dead. if a squadron got killed, it's gone and while that works pretty much as intended, as long as AI flights are this unpredictable, it would be very nice to have the option to just revive dead pilots, maybe after checking a box in the cheat options.

EDIT: maybe the two buttons, that normally are used to switch a pilot from AI to player und to put them on leave can be converted for that when pilots are dead, since in that case they both just read "pilot dead".

**Comment by DanAlbert** [comment]
You can change the replenishment rate. Are you losing more than 20 pilots per squadron per turn?

**Comment by dasrehzockt** [comment]
no, but with limited funds, losing 2-6 pilots in total (which can be quite normal in most of my scenarios) per turn can have the same effect in a short amount of rounds. AI does not like to play with high digit sams and IADS even though the latest loadout changes seemed to help alot with DEAD! 

so i tried to look at the documentation, and it mentions a replenishment rate of 1 pilot per squadron (every turn i assume). however i did not find any info on how to change that value and the only way i see to get more pilots per turn, is to have more squadrons in total. so right now i got it packed up to 6 hornet squadrons, which should be more than enough to compensate the unneccesary losses. 
however, this could also be a problem. wouldn't it be better in terms of balancing, if this was a set value for all squadrons, comparable to income and you can alter it by a multiplier in the UI? in that case, the amount of pilots per turn can change over the course of the campaign, e.g. for every captured base. 
and maybe handle this like buying planes and ground units, which would help if i got a lot of squadron templates going on. i could choose how many pilots go to each squadron, instead of packing the first one full, which might be set to A2A only, leaving me with no A2G pilots. Deactivating each pilot by hand is kind of tedious atm. (would be great if i could select multiple pilots and change these all at once btw. )

the ability to have this value either static or scalable would be great and the ability to turn this feature off entirely would be a nice to have, even though i highly endorse the addition of dead pilots!^^

however, i still think, that just havin a button next to a dead pilot to get him back to life would be really neat, just for sakes of choosing who is gonna be able to fly with me next round.

**Comment by DanAlbert** [comment]
![image](https://user-images.githubusercontent.com/315852/122656661-891d9a80-d111-11eb-9762-753856a1bb7f.png)

**Comment by dasrehzockt** [comment]
i'm currently on version 3.1.0-1962 and do not have these options. the wiki mentions this as a 4.0 feature as well. am i missing sth here?

EDIT: ok i've jumped to a more current preview build to check if i'm just on version that hasn't got this feature yet. however, i could not test cuz of issue #1286

EDIT2: ok, #1286 is out of the way and yes, now i can use the options from the screenshot you posted. however, the pilots per squadron cannot be lower than 12. i would like this value to be not capped in that direction, since i'm using the squadrons primarily to make use of different liveries.
besides that, i can live with how it is for now, even though i still endorse the idea of a faction based value that can change over the course of a campaign, or is static, depending on what faction you play maybe. having this replenishment rate per squadron can cause problems depending on how many you have going on at the same time. also it would be wise, to add the possibility for less than one pilot per turn, so it's maybe one every four turns for example.

**Comment by dasrehzockt** [comment]
would it be possible to have the maximum number of pilots per squadron be less than 12 and the pilots per round less than 1? right now i'm working around having too many pilots in a squadron by sending pilots on leave manually.

Also i would like to pitch the idea of moving the replenishment rate for each squadron to the .yaml files for each squadron. so if you tick the 'enable per squadron' button, it uses the values you set in each squadron file and if not it uses the global value. 
that way we can have squadrons replenish at different rates.

i know this is still WIP but changing the min and max values should be an easy task i assume.
